### ISSUE TITLE
Caches with modified listing coords parsed as mystery on live map

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#6714]
Some weeks ago gc.com improved their live map (with feature c:geo already had before :) ).

If a cache has modified listing coords (solution stored by user) it will be shown as a blue puzzle piece icon instead of the original cache type icon.

This results in the cache being parsed as mystery or ?? on our live map (fast mode).

I assume this is rarely recognized by users as most of them will have their solved caches also stored in c:geo locally (which overrides any wrongly parsed results).

**Comment by Lineflyer** [comment]
Obsolete after #7514 was merged.

A separate point for a new issue is, that solved mysteries could be loaded as such.
